typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,19 @@
+let lightSwitch = false;
+
+const $bulb = document.querySelector('.bulb');
+const $bulbBackground = document.querySelector('.bulb-background');
+if (!$bulb) throw new Error('$bulb query has failed');
+$bulb.addEventListener('click', clickHandler);
+function clickHandler(): void {
+  if (!$bulbBackground) throw new Error('$bulbBackground query has failed');
+  if (!$bulb) throw new Error('$bulb query has failed');
```

Comment:
These checks for if the query failed can be outside of this callback function because we just need it to be not null when we start working with it.